
<launch>
<!--启动一个usb相机，需要增加对应修改：usb_cam_read_yyx/src/usb_cam-develop/CMakeLists.txt和usb_cam_read_yyx/src/usb_cam-develop/nodes/目录下文件-->
    <!-- Node to start USB camera -->
    <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen">
        <!-- Configure USB camera parameters here -->
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="1280" />
        <param name="image_height" value="720" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="camera" />
    </node>
 
</launch>
